llvm/include/llvm/CodeGen/MachineInstr.h

```diff
@@ -25,6 +25,7 @@
 #include "llvm/CodeGen/TargetOpcodes.h"
 #include "llvm/IR/DebugLoc.h"
 #include "llvm/IR/InlineAsm.h"
+#include "llvm/IR/MemoryModelRelaxationAnnotations.h"
```

Comment:
In this and other headers, do we really need to include this header?

Reply:
nope, another leftover of previous design iterations :(